### DISCUSSION TITLE
Sigma symbol is not getting rendered at correct position

### DISCUSSION THREAD

**Discussion by vikassnwl** [community/community#133708] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

![Screenshot from 2024-07-27 11-32-47](https://github.com/user-attachments/assets/ed748c85-bdaa-4dbd-a98a-d30fd1c7211f)
![Screenshot from 2024-07-27 11-32-34](https://github.com/user-attachments/assets/baadd7cf-9070-4045-923c-23d2e523f9b6)

As you can see in the above screenshot that how the sigma symbol is getting rendered at incorrect position.
It's happening only on GitHub. If I use the same markdown syntax on Jupyter Notebook, I get the expected result as shown in the following screenshot.
![Screenshot from 2024-07-27 11-47-43](https://github.com/user-attachments/assets/6b72f7c3-1497-456d-bd7a-9e44ce7f165a)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Buddhi19** [comment]
Try,
"$$\sum_{i=1}^n$$"
$$\sum_{i=1}^n$$

**Reply by vikassnwl** [reply]
Sorry, I forgot to mention inline formula rendering.
I have issue with Sigma symbol rendering with inline formulas i.e. one enclosed within single $ symbol.

**Reply by Buddhi19** [reply]
You can try this,
"${\sum}^n_{i=1}$"
${\sum}^n_{i=1}$

**Comment by Buddhi19** [comment, marked as answer]
Will add the reply here as well,
"${\sum}^n_{i=1}$"
${\sum}^n_{i=1}$

**Reply by vikassnwl** [reply]
This will also work:

"${\sum}_{i=1}^n$"

${\sum}_{i=1}^n$

The issue was that I was not enclosing \sum withing curly braces.

Thanks for helping me figure it out. 😊
